js/activesync/account.js

```diff
@@ -368,7 +370,7 @@ ActiveSyncAccount.prototype = {
   },
 
   shutdown: function asa_shutdown(callback) {
-    this._LOG.__die();
+    logic(this, 'die');
```

Comment:
Yes, that definitely could be misread and I chuckled to myself the first time I saw it in the logs. I'll remove them for now. (I considered renaming them in case we could use them to spot future regressions, but since we don't have any tests which care about this, it's just noise for now.)